WebDevStudios/create-block
# Feature/20 Update to Sync with v1.1.1

Closes #20 

### DESCRIPTION ###
Update the `create-block` to use `WebDevStudios/WDS Block Starter` v1.1.1

### SCREENSHOTS ###
<img width="1510" alt="Screen Shot 2020-08-21 at 11 30 41 PM" src="https://user-images.githubusercontent.com/5747475/90908002-609c1580-e406-11ea-87ec-fe25579529b2.png">

### STEPS TO VERIFY ###
1. Follow this wiki to test this PR's branch - https://github.com/WebDevStudios/create-block/wiki/Using-or-Testing-a-branch
1. Scaffold a plugin and wait for it end successfully.
1. The scaffolded plugin should work as expected.
